packages/cli-lib/examples/slack/README.md

```diff
@@ -0,0 +1,30 @@
+## Slack Integration Example
```

Comment:
This is a pretty neat example, although I wonder if we would want to recommend doing this via a separate step in a GitHub action since it will be less fragile. We also could incorporate into https://github.com/HubSpot/hubspot-cms-deploy-action to make it easy for folks.

Reply:
Yeah that's why I mentioned in the overview below it would be likely be invoked from a CI/CD platform such as Jenkins or Github.  Are you suggesting I build the example off of the deploy github action?

Reply:
I was thinking that we add Slack to the https://github.com/HubSpot/hubspot-cms-deploy-action with a mechanism to configure it, if possible. I felt like that while cool—this example was suggesting a less-than-ideal solution.

Reply:
I can write up something to add it to the deploy action